### DISCUSSION TITLE
Help with RIME not displaying correct comment

### DISCUSSION THREAD

**Discussion by vienvu1** [rime/home#1393] [General]
I'm creating a Sino-Vietnamese input on my end and everything seems to work well except for one entry not displaying my comment "漢越字“.

Here is the code entry in my yaml file.
![lễ](https://github.com/rime/home/assets/148301222/bf624a14-532d-44bd-af2d-bb51efdfe32c)

Here is how it displays in RIME

![lễ2](https://github.com/rime/home/assets/148301222/00c41ce6-c5f7-47fa-8f72-168ff90f8fe7)

Is there a way to fix this? It looks like the only entry having this issue,

**Comment by fxliang** [comment]
you should upload your configuarions for developers to test.

**Reply by vienvu1** [reply]
I've just tested on another computer and it appears to be working. I think it might be a caching issue. Is there a way to clear caching?
